addon/adapters/application.js

```diff
@@ -128,7 +128,6 @@ export default Ember.Object.extend(FetchMixin, Evented, {
     // use resource's service if in container, otherwise use this service to fetch
     let owner = (typeof getOwner === 'function') ? getOwner(this) : this.container;
     let service = owner.lookup('service:' + pluralize(type)) || this;
-    url = this.fetchUrl(url);
```

Comment:
(updateRelationship calls patchRelationship](https://github.com/pixelhandler/ember-jsonapi-resources/blob/master/addon/mixins/resource-operations.js#L136) which calls [fetch](https://github.com/pixelhandler/ember-jsonapi-resources/blob/master/addon/adapters/application.js#L281)

Even if updateRelationship's stack did not call `fetch`, I removed `fetchUrl` from `findRelated`, which is unrelated to `updateRelationship` or `patchRelationship`. 

But do check to be sure.